README.md

```diff
@@ -1,2 +1,54 @@
-# rivendell-ah-client
+# rivendell-ah-client Author's Haven
 [![CircleCI](https://circleci.com/gh/andela/rivendell-ah-client/tree/staging.svg?style=svg)](https://circleci.com/gh/andela/rivendell-ah-client/tree/staging)
+
+Authors Haven is am online application that bring together a community of authors and readers. It allows Authors to create and post articles based on topic of interest or popular demands. Reader can search for articles to read. Readers can also follow authors if they like their articles. Other features supported by the application includes: 
```

Comment:
correct typo  on  description it should be `Author's haven is 'an' online application that 'brings' together a community of authors .....It allows 'authors' (lowercase a)...... based on 'their' or popular 'demand' (singular). 'Readers' (plural)`